listenbrainz_spark/stats/user/tests/test_listening_activity.py

```diff
@@ -78,7 +79,7 @@ def test_get_listening_activity_year(self, mock_create_messages, _, mock_get_lis
         mock_create_messages.assert_called_with(data='activity_table', stats_range='year',
                                                 from_date=from_date, to_date=to_date)
 
-    @patch("listenbrainz_spark.stats.user.listening_activity.get_latest_listen_ts")
+    @patch("listenbrainz_spark.stats.common.listening_activity.get_latest_listen_ts")
```

Comment:
Should this test test the other time_ranges as well? I see tests for only quarter and half_yearly. These date function always scare me, so more tests could really be appropriate here.

Reply:
Added tests for all ranges.